Tools/MGCB/ConsoleLogger.cs

```diff
@@ -11,13 +11,13 @@ public class ConsoleLogger : ContentBuildLogger
     {
         public override void LogMessage(string message, params object[] messageArgs)
         {
-			Console.WriteLine(message, messageArgs);
+			Console.WriteLine(IndentString + message, messageArgs);
```

Comment:
ok, so because most (all?) content exceptions are caught outside the `_manager.BuildContent()` the _manager.Logger is allready Unindented by the Finally block. No way to test this.
Need a bit of refactoring.
I tried with an invalid .spritefont and an .fbx.